website/blog/2022-12-28-databend-weekly-74.md

```diff
@@ -0,0 +1,179 @@
+
+**Planner**
+
+- support selectivity estimation for range predicates ([#9398](https://github.com/datafuselabs/databend/pull/9398))
+
+**Query**
+
+- support copy on error ([#9312](https://github.com/datafuselabs/databend/pull/9312))
+- support databend-local ([#9282](https://github.com/datafuselabs/databend/pull/9282))
+- external storage support location part prefix ([#9381](https://github.com/datafuselabs/databend/pull/9381))
+
+**Storage**
+
+- rangefilter support in ([#9330](https://github.com/datafuselabs/databend/pull/9330))
+- try to improve object storage io read ([#9335](https://github.com/datafuselabs/databend/pull/9335))
+- supprot table compression ([#9370](https://github.com/datafuselabs/databend/pull/9370))
+
+**Metrics**
+
+- add more metrics for fuse compact and block write ([#9399](https://github.com/datafuselabs/databend/pull/9399))
+
+**Sqllogictest**
+
+- add no-fail-fast support ([#9391](https://github.com/datafuselabs/databend/pull/9391))
+
+### Code Refactoring :tada:
+
+**\***
+
+- adopt rustls entirely, removing all deps to native-tls ([#9358](https://github.com/datafuselabs/databend/pull/9358))
+
+**Format**
+
+- remove format_xxx settings ([#9360](https://github.com/datafuselabs/databend/pull/9360))
+- adjust interface of FileFormatOptionsExt ([#9395](https://github.com/datafuselabs/databend/pull/9395))
+
+**Planner**
+
+- remove `SyncTypeChecker` ([#9352](https://github.com/datafuselabs/databend/pull/9352))
+
+**Query**
+
+- split fuse source to read data and deserialize ([#9353](https://github.com/datafuselabs/databend/pull/9353))
+- avoid io copy in read parquet data ([#9365](https://github.com/datafuselabs/databend/pull/9365))
+- add uncompressed buffer for parquet reader ([#9379](https://github.com/datafuselabs/databend/pull/9379))
+
+**Storage**
+
+- add read/write settings ([#9359](https://github.com/datafuselabs/databend/pull/9359))
+
+### Bug Fixes :wrench:
+
+**Format**
+
+- fix align_flush with header only ([#9327](https://github.com/datafuselabs/databend/pull/9327))
+
+**Settings**
+
+- use logical CPU number as default value of num_cpus ([#9396](https://github.com/datafuselabs/databend/pull/9396))
+
+**Processors**
+
+- the data type on both sides of the union does not match ([#9361](https://github.com/datafuselabs/databend/pull/9361))
+
+**HTTP Handler**
+
+- false alarm (warning log) about query not exists ([#9380](https://github.com/datafuselabs/databend/pull/9380))
+
+
+**Sqllogictest**
+
+- refactor sqllogictest http client and fix expression string like ([#9363](https://github.com/datafuselabs/databend/pull/9363))
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+#### Support Databend-Local
+
+Like [clickhouse-local](https://clickhouse.com/docs/en/operations/utilities/clickhouse-local/), databend-local can perform fast processing on local files without launching a databend cluster.
```

Comment:
Inspired by [clickhouse-local](https://clickhouse.com/docs/en/operations/utilities/clickhouse-local/), databend-local allows you to perform fast processing on local files, without the need of launching a Databend cluster.